azure-pipelines.yml

```diff
@@ -6,11 +6,11 @@ jobs:
     matrix:
       Ubuntu1804:
         os: 'ubuntu:18.04'
-        command: './lint.sh'
+        command: './test.sh'
```

Comment:
The idea is that command can be `build.sh && test.sh` for most OSs and then it can be `lint.sh && build.sh && test.sh` for the most up to date OS.